app/services/sweeper.py

```
@@ -0,0 +1,241 @@
+                    )
+                    try:
+                        async with db.begin():
+                            record.status = EmailStatus.FAILED.value
+                            record.error_message = failure_reason
+                    except Exception as e:
+                        self._errored_total += 1
+                        sweeper_errored_total.inc()
+                        try:
+                            record.error_message = f"{failure_reason}; persist error: {e}"
+                        except Exception:
+                            pass
```

Ruff (0.14.10)

`try`-`except`-`pass` detected, consider logging the exception

(S110)